libmscore/rest.cpp

```diff
@@ -108,8 +118,14 @@ void Rest::draw(QPainter* painter) const
             painter->drawLine(QLineF(0.0, -_spatium, 0.0, _spatium));
             painter->drawLine(QLineF(x2,  -_spatium, x2,  _spatium));
 
+            // draw number
             std::vector<SymId>&& s = toTimeSigString(QString("%1").arg(n));
```

Comment:
> 
> 
> > Yay! What would be useful (and not difficult to implement I believe) would be the interval based measure numbers.
> > ![image](https://user-images.githubusercontent.com/35939574/76655391-88c44600-6543-11ea-9d1f-46a32116168b.png)
> > I believe that there is an issue associated with it, but I can't find it.
> 
> That would be part of a Google Summer of Code Project I am considering. :-)

Wouldn't it just come here? replacing that string?

Actually, do you mind if I PR your branch?

Reply:
Actually, it is more complicated than that, so I'll leave it to you GSoC 😜